sdk/containerregistry/container-registry/samples/javascript/containerRegistryClient.js

```diff
@@ -10,7 +10,7 @@ const { DefaultAzureCredential } = require("@azure/identity");
 require("dotenv").config();
 
 async function main() {
-  const endpoint = process.env.ENDPOINT ?? "<endpoint>";
+  const endpoint = process.env.ENDPOINT || "<endpoint>";
```

Comment:
@jeremymeng Is this change still needed now that wer're running samples in Node 10? We'd like to be able to use new syntax like this for samples.

Reply:
It is still causing failure in current pipeline https://dev.azure.com/azure-sdk/internal/_build/results?buildId=793325&view=logs&j=f638a4bc-e47f-5332-e3d6-7d060d0151e6&t=6d2f55a7-d523-599a-86b1-004671b71436